bcipy/gui/bciui.py

```diff
@@ -0,0 +1,324 @@
+import os
+from typing import Callable, Type
+from PyQt6.QtCore import pyqtSignal
+from PyQt6.QtWidgets import (
+    QWidget,
+    QApplication,
+    QVBoxLayout,
+    QLabel,
+    QHBoxLayout,
+    QPushButton,
+    QComboBox,
+    QScrollArea,
+    QLineEdit,
+    QLayout,
+    QSizePolicy,
+    QMessageBox,
+)
+from typing import Optional, List
+
+
+class BCIUI(QWidget):
+    contents: QVBoxLayout
+    center_content_vertically: bool = False
+
+    def __init__(self, title="BCIUI", default_width=500, default_height=600):
+        super().__init__()
+        self.resize(default_width, default_height)
+        self.setWindowTitle(title)
+        self.contents = QVBoxLayout()
+        self.setLayout(self.contents)
+
+    def app(self): 
```

Comment:
Needed?

Reply:
Maybe I should make this an abstract method

Reply:
Seems like I can't inherit from both `QWidget` and `ABC` so leaving this in for now